docs/source/index.rst

```diff
@@ -52,6 +52,16 @@ model implementations and application components.
    tutorials/mvdr_tutorial
    tutorials/hybrid_demucs_tutorial
 
+.. toctree::
+   :maxdepth: 1
+   :caption: Training Recipes
+   :hidden:
+
+   Conformer RNN-T ASR <https://github.com/pytorch/audio/tree/main/examples/asr/librispeech_conformer_rnnt>
+   Emformer RNN-T ASR <https://github.com/pytorch/audio/tree/main/examples/asr/emformer_rnnt>
+   ConvTas-Net Source Separation <https://github.com/pytorch/audio/tree/main/examples/source_separation>
```

Comment:
nit: people usually call it "Conv-TasNet" or "ConvTasNet"
```suggestion
   Conv-TasNet Source Separation <https://github.com/pytorch/audio/tree/main/examples/source_separation>
```

Reply:
oops, that's what I meant. Fixed